### PR TITLE
Proposal Airbnb Stylelint Config

### PR DESCRIPTION
**Pull request by Drapegnik**
* add [`airbnb/css`](https://github.com/airbnb/css) via [config gist](https://gist.github.com/DanielaValero/b3a72b2f3d0c85c1985d5d2d7497f347)
* add [`prettier`](https://github.com/prettier/prettier) with [`prettier-stylelint`](https://github.com/hugomrdias/prettier-stylelint)

Please use with 
* [`linter-stylelint`](https://atom.io/packages/linter-stylelint)
* [`prettier-eslint`](https://atom.io/packages/prettier-eslint)

or your editor analogs

### CODE REVIEW

**Comment by UladBohdan** [comment]
@Drapegnik what about this PR ?

**Comment by Drapegnik** [comment]
@andemerie, please review config, especially is [`stylelint-selector-bem-pattern`](https://github.com/simonsmith/stylelint-selector-bem-pattern) working?

**Comment by UladBohdan** [comment]
Overall it works fine; no warnings in start.

@Drapegnik probably you could fix that:
`[22:29:02] Server stopped`
`[22:29:02] The following tasks did not complete: watch, <parallel>, serve, livereload, sass:watch, ejs:watch`
`[22:29:02] Did you forget to signal async completion?`
we have it for a long time already. It is so annoying.

File: src/about/about.ejs [hunk 1]
```diff
@@ -1,144 +1,146 @@
...
     </div>
-</div>
+  </div>
 
-<% include ../footer/footer.ejs %>
+  <% include ../footer/footer.ejs %>
 </body>
-</html>
+
+</html>
```
Comment (andemerie) [inline src/about/about.ejs]: There are a lot of file without empty lines at the end. Could you please add them?

**Pull request merged**